proxy/src/main/java/com/wavefront/agent/logsharvesting/EvictingMetricsRegistry.java

```diff
@@ -6,13 +6,7 @@
 import com.github.benmanes.caffeine.cache.Caffeine;
 import com.github.benmanes.caffeine.cache.LoadingCache;
 import com.wavefront.agent.config.MetricMatcher;
-import com.yammer.metrics.core.Counter;
-import com.yammer.metrics.core.Gauge;
-import com.yammer.metrics.core.Histogram;
-import com.yammer.metrics.core.Metric;
-import com.yammer.metrics.core.MetricName;
-import com.yammer.metrics.core.MetricsRegistry;
-import com.yammer.metrics.core.WavefrontHistogram;
+import com.yammer.metrics.core.*;
```

Comment:
we dont encourage wild card imports.
use the same .idea/codeStyleSettings.xml present in the system repo to avoid this?

Reply:
Fixed.